accepted/future-releases/enhanced-enums/feature-specification.md

```diff
@@ -102,35 +105,33 @@ The semantics of such an enum declaration, *E*, is defined as introducing a (sem
-  
-- their name `id` as a string `"id"`, 
+
+  - their name `id` as a string `"id"`, 
   - their index *i* as an `int`, and
   - their `enum` class’s name as a string, `"Name"`,
-  
-  all of which is accessible to the `toString` and `index` member of `Enum`, and to the `EnumName.name` extension getter. The values are computed as follows.
-  
-  - `id` &mapsto; `Name()` (no arguments, equivalent to empty argument list)
-  - `id(args)` &mapsto; `Name(args)`
-  - `id<types>(args)` &mapsto; `Name<types>(args)`
-  - `id.named(args)` &mapsto; `Name._$named(args)`
-  - `id<types>.named(args)` &mapsto; `Name<types>._$named(args)`
-  
+
+  all of which are accessible to the `toString` and `index` member of `Enum`, and to the `EnumName.name` extension getter. The values are computed as by the following constant constructor invocations.
+
+  - `id` &mapsto; `const Name()` (no arguments, equivalent to empty argument list)
+  - `id(args)` &mapsto; `const Name(args)`
+  - `id<types>(args)` &mapsto; `const Name<types>(args)`
+  - `id.named(args)` &mapsto; `const Name.named(args)`
+  - `id<types>.named(args)` &mapsto; `const Name<types>.named(args)`
+
   where `args` are considered as occurring in a `const` context, and it’s a **compile-time error** if they are then not compile-time constants.
-  
-  Type inference is applied to the resulting constructor invocations, with no context type, where necessary, so omitted type arguments to a generic `enum` class are filled in by type inference, using the type of arguments, if any, and then the type of the constant variable is the static type of the constant object creation expression.
-  
+
+  Type inference, with no context type, is applied to the resulting constructor invocations where necessary, so omitted type arguments to a generic `enum` class are filled in by type inference, using the type of arguments, if any, and then the type of the constant variable is the static type of the constant object creation expression.
+
   The objects created here are *not canonicalized* like other constant object creations. _(In practice, the index value is considered part of the object, so no two objects will have the same state.)_
-  
-- **Static `values` list**: If the class does not declare or inherit a member with base-name `values`, a static constant variable named `values` is added as by the declaration  `static const List<Name> values = [id1, …, idn];`
+
+- **Static `values` list**: A static constant variable named `values` is added as by the declaration  `static const List<Name> values = [id1, …, idn];`
   where `id1`…`idn` are the names of the enum entries of the `enum` declaration in source/index order.
   If `Name` is generic, the `List<Name>` instantiates `Name` to its bounds.
```

Comment:
I don't think so. It's not a given.

Reply:
I think it is a given. Otherwise, when `Name` is generic, what could `List<Name>` mean, other than `List<Name<T1, .. Tk>` where `T1, .. Tk` are chosen by instantiation to bounds?

Reply:
ACK, the actual code is given above, and that would trivially use instantiate to bounds.